.history/public/index_20230327085330.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../styles/style.css">
+    <title>Ayomide S.E.P</title>
+</head>
+<body>
+    
+    <header class="header header-top">
+        <div class="header-inner sp-btw-flex">
+            <div class="heoader-log">
+                <img src="../assets/Logo Placeholder.png" alt="">
+            </div>
+            <div class="header-mobile-icon">
+                <img src="../assets/Normal Button/Tertiary/Enabled.png" alt="">
+            </div>
+        </div>
```

Comment:
- [ ] Nice work dear dev, but I think you would do better changing this div on line 16 into a nav tag instead as per the requirement, this is for semantic reason, the header items have to be wrapped in a nav tag for SEO performace